### PR TITLE
add whoami workflow [HEAD-35]

### PR DESCRIPTION
**Pull request by j-luong**
**What does this PR do?**
This PR should introduce a new localworkflow hidden behind the experimental flag, it should allow the CLI to call the user/me endpoint.

### CODE REVIEW

File: pkg/local_workflows/whoami_workflow.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+	// return full payload if json flag is set
+	if config.GetBool(jsonFlag) {
+		// parse response
+		userMeData := createWorkflowData(userMe)
+
+		// return userme data
+		return []workflow.Data{userMeData}, err
+	}
+
+	userData := createWorkflowData(user)
```
Comment (PeterSchafer) [inline pkg/local_workflows/whoami_workflow.go:83]: In this case the mimetype should be `plain/text` rather than `application/json`.

Comment (j-luong) [reply pkg/local_workflows/whoami_workflow.go]: good catch, fixed :)

File: pkg/local_workflows/whoami_workflow_test.go [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+	t.Run("returns user name", func(t *testing.T) {
+		// setup
+		expectedResponse := "user.name@snyk.io"
+
+		// execute
+		output, err := whoAmIWorkflowEntryPoint(invocationContextMock, nil)
+
+		// assert
+		assert.Nil(t, err)
+		assert.Equal(t, expectedResponse, output[0].GetPayload())
```
Comment (PeterSchafer) [inline pkg/local_workflows/whoami_workflow_test.go:115]: Suggestion: Maybe test the mime-type here and below.

File: pkg/local_workflows/whoami_workflow_test.go [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+	assert.Equal(t, expectedError.Error(), err.Error())
+}
+
+func Test_WhoAmI_whoAmIWorkflowEntryPoint_happyPath(t *testing.T) {
+	// setup
+	logger := log.New(os.Stderr, "test", 0)
+	config := configuration.New()
+	config.Set("experimental", true)
+
+	payload := `{
```
Comment (PeterSchafer) [inline pkg/local_workflows/whoami_workflow_test.go:65]: nitpick: from a maintainability aspect it is good not to share knowledge if not necessary, this means you could consider using the api contract data structure to create the json payload.

**Review by PeterSchafer — APPROVED** [review]
Looks great!
Same as before, please squash before merging :)

**Pull request merged**